### ISSUE TITLE
Change?

### ISSUE THREAD

**Issue by PeterKaa1** [radimitrov/CSharpShellApp#338]
In the assemblies folder of apk there is no longer a dll called AppAssembly, now there is an dll which has the same name as the project and also an pdb file is there. Did you add this in the latest update?

**Comment by radimitrov** [comment]
Yes. File name had to be the same as assembly name for said pdb logic. Shouldn't have any other effect

**Comment by PeterKaa1** [comment]
> Yes. File name had to be the same as assembly name for said pdb logic. Shouldn't have any other effect

Thanks for the quick reply, what is CrashSharpener? Also, what is the purpose of pdb in .NET MAUI projects?

**Comment by radimitrov** [comment]
[PDB](https://learn.microsoft.com/en-us/visualstudio/debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger?view=vs-2022) is for debugging, or in the context of C# Shell - showing on which line and in which source file the Exception was thrown and only the method name. CrashSharpener is the "glue" between them. Crash notifications and PDB inclusion is on by default in the APK settings.
